edx-platform/pearson-ukl-theme/lms/templates/static_templates/privacy.html

```diff
@@ -1,177 +0,0 @@
-## mako
```

Comment:
Are Privacy and Tos not necesary in the UKL theme? I mean doesn't this site have those pages or those were replaced with other ones?

Reply:
Those links are added through the Django admin and it's not necessary to have these pages.